totalRP3/Core/Utils.lua

```diff
@@ -735,18 +733,6 @@ local function safeDeserialize(structure, default)
 end
 Utils.serial.safeDeserialize = safeDeserialize;
 
-Utils.serial.decompressCodedStructure = function(message)
-	return deserialize(libCompress:Decompress(libCompressEncoder:Decode(message)));
-end
-
-Utils.serial.safeDecompressCodedStructure = function(message)
-	return safeDeserialize(libCompress:Decompress(libCompressEncoder:Decode(message)));
-end
-
-Utils.serial.encodeCompressStructure = function(structure)
-	return libCompressEncoder:Encode(libCompress:Compress(serialize(structure)));
-end
```

Comment:
Two of these were unused, and one only had the single usage in RegisterExchange.lua - so killing all three.

Aside - I want to kill the other Utils.serial functions, but they're currently being used in Extended until that migrates to using the new export/import format.